jwst/ami/utils.py

```diff
@@ -1411,7 +1406,7 @@ def get_cw_beta(bandpass):
     return cw, beta
 
 
-def cdmatrix_to_sky(vec, cd11, cd12, cd21, cd22):
+def _cdmatrix_to_sky(vec, cd11, cd12, cd21, cd22):
```

Comment:
As this function is only used within this module (and only within `degrees_per_pixel`) I added an underscore prefix to denote that it is a 'private' function that outside code should not use.